boards/cogip-nucleo-f446re/doc.txt

```diff
@@ -0,0 +1,73 @@
+/**
+@defgroup    boards_nucleo-f446re STM32 Nucleo-F446RE
+@ingroup     boards_common_nucleo64
+@brief       Support for the STM32 Nucleo-F446RE
```

Comment:
Maybe we should change this too to something custom. Have you try 'make doc' ?

Reply:
Done.
For 'make doc', I think some 'doxygen.txt' are missing in 'applications', 'boards', 'examples', 'lib and 'sys', but not sure it should be part of this PR.

Reply:
Yes I know there is so much documentation to still write. Thats a huge work.
What I mean is that it is a good practice to test 'make doc' on new/refactored code at least to check if sections are well ordered.